src/styles/App.css

```diff
@@ -1,5 +1,80 @@
 * {
-  margin: 0;
+  margin: 10px;
+  list-style: none;
+  text-decoration: none;
   padding: 0;
   box-sizing: border-box;
 }
+
+#root {
+  margin: 0;
+  padding: 0 0 8%;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgb(0 0 0 / 5%);
+  background-color: #fafafa;
+  min-height: 100vh;
+}
+
+.input-area {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.imput-cont {
+  display: flex;
+  justify-content: space-between;
+}
+
+button {
+  cursor: pointer;
+  border: none;
+  background-color: #0290ff;
+  border-radius: 3px;
+  padding: 8px 24px;
+  font-family: "Roboto Slab", serif;
+  font-weight: 300;
+  letter-spacing: 0.5px;
+  color: #fff;
+}
+
+input {
+  padding: 12px;
+  border-radius: 4px;
+  border: 1px solid #e8e8e8;
+  background-color: #fff;
+  font-size: 1rem;
+  letter-spacing: -0.15px;
+  color: #121212;
+}
+
+.add-book {
+  font-size: 1.25rem;
+  font-weight: 700;
+  letter-spacing: -0.18px;
+  color: #888;
+}
```

Comment:
- [ ] The input field, select option for author and the button for adding book do not match the original design styles. The styles for your input fields and the button does not look nice or as close to the original design given and it's a requirement for your styles to match the given design. Kindly style the input field for the book title, author select option, and button so they look like the template design given. 👍🏽

<img width="1399" alt="Screenshot 2023-02-04 at 8 23 43 PM" src="https://user-images.githubusercontent.com/50941074/216788612-b3b666a4-941c-48ce-98d0-6a6809ebdee2.png">